webpack/assets/javascripts/react_app/stores/HostsStore.js

```diff
@@ -0,0 +1,46 @@
+import AppDispatcher from '../dispatcher';
+import {ACTIONS} from '../constants';
+import AppEventEmitter from './AppEventEmitter';
+
+// Internal object of Hosts
+const _hosts = {};
+
+class HostsEventEmitter extends AppEventEmitter {
+  constructor() {
+    super();
+  }
+
+  getHostsData(id) {
+    _hosts[id] = _hosts[id] || { data: [] };
+
+    return _hosts[id];
+  }
+}
+
+const HostsStore = new HostsEventEmitter();
+
+HostsStore.setMaxListeners(50);
```

Comment:
Might this not have an issue if someone sets the table size to more then 50 rows?